ivy/functional/backends/jax/gradients.py

```diff
@@ -145,10 +145,15 @@ def stop_gradient(
 
 
 def jac(func: Callable):
-    grad_fn = lambda x_in: ivy.to_native(func(x_in), nested=True)
+    grad_fn = lambda x_in: ivy.to_native(
+        func(ivy.to_ivy(x_in, nested=True)),
+        nested=True,
+        include_derived={tuple: True},
+    )
     callback_fn = lambda x_in: ivy.to_ivy(
-        jax.jacfwd(grad_fn)((ivy.to_native(x_in, nested=True))),
+        jax.jacrev(grad_fn)((ivy.to_native(x_in, nested=True))),
         nested=True,
+        include_derived={tuple: True},
```

Comment:
Good catch! `include_derived=True` would be even more generalized in that case, so that we cover other derived types like `ivy.Container`. Same applies to the other backends